### PR TITLE
dev version of ggforest function

### PR DESCRIPTION
**Pull request by pbiecek**
A forest plot, as suggested in #114
Right now it has only one panel with confidence intervals,
will add some numerical statistics later

### CODE REVIEW

**Comment by kassambara** [comment]
<pre>
R CMD check results
0 errors | 0 warnings | 1 note 

checking R code for possible problems ... NOTE
ggforest: no visible binding for global variable ‘term’
ggforest: no visible binding for global variable ‘estimate’
ggforest: no visible binding for global variable ‘issig’
ggforest: no visible binding for global variable ‘conf.low’
ggforest: no visible binding for global variable ‘conf.high’
ggforest: no visible binding for global variable ‘p.val’
ggforest: no visible binding for global variable ‘.x’
Undefined global functions or variables:
  .x conf.high conf.low estimate issig p.val term
</pre>

File: DESCRIPTION [hunk 1]
```diff
@@ -24,13 +25,14 @@ Imports:
...
     stats,
+    broom,
     tidyr
 Suggests:
     knitr,
     RTCGA.clinical(>= 20151101.4.0),
     survMisc
 VignetteBuilder: knitr
-biocViews:
+biocViews:
```
Comment (MarcinKosinski) [inline DESCRIPTION]: @kassambara @pbiecek this section `biocViews`  is empty and is only used at Bioconductor. I suggest this should be removed

Comment (kassambara) [inline DESCRIPTION]: I agree with you! Thanks!

File: R/ggforest.R [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+#' @param xlab Label in OX axis.
+#' @param font.main,font.submain,font.caption,font.x,font.y,font.tickslab a vector of length 3
+#'  indicating respectively the size (e.g.: 14), the style (e.g.: "plain",
+#'  "bold", "italic", "bold.italic") and the color (e.g.: "red") of main title, subtitle, caption,
+#'  xlab and ylab and axis tick labels, respectively. For example \emph{font.x =
+#'  c(14, "bold", "red")}.  Use font.x = 14, to change only font size; or use
+#'  font.x = "bold", to change only font face.
+#'
+#' @return return an object of class ggplot
+#'
```
Comment (MarcinKosinski) [inline R/ggforest.R]: @pbiecek you can also add `authors` section with your credentials

**Pull request merged**